src/pages/condor/setting-up-condor-local.md

```diff
@@ -28,44 +28,108 @@ NCTL is your tool for managing the Casper network. We'll use a Dockerized versio
    cd casper-nctl-docker
    ```
 
-2. **Switch to the Condor Branch:**
+2. **Switch to the 2.0 Branch:**
    ```bash
    git checkout feat-2.0
```

Comment:
Is this even still accurate? Why not just `dev` or the casper 2.0.0 release tag?

Reply:
It will cross check with @Jiu-hong who updated the content originally.

Reply:
Hi @mssteuer , I received the below clarification from @Jiu-hong ;

> That is https://github.com/make-software/casper-nctl-docker 
>
> which doesn't have `dev` or `2.0.0` tag
> 
> ![image](https://github.com/user-attachments/assets/9a74eb07-efee-44ed-8f9c-84701a4c0fa8)
> 
> `master` is `1.5` and `feat-2.0` is the most recent one